microsoft-edge/progressive-web-apps-chromium/how-to/webappmanifests.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Use a Web App Manifest to integrate a Progressive Web App into the Operating System
+description: Learn how to use the Web App Manifest to integrate your Progressive Web App into your operating system.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 01/07/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: pwa
+keywords: progressive web apps, PWA, Edge, JavaScript, Windows, UWP, Microsoft Store
+---
+# Use a Web App Manifest to integrate a Progressive Web App into the Operating System
+
+A Web App Manifest of a website governs how your Progressive Web App (PWA) looks and behaves when installed on a device. The Web App Manifest provides information such as the name of your app, the file location of icons that represent your app in system menus, and the theme colors that the operating system (OS) uses in the title bar.
+
+A Web App Manifest is a JSON file that must be referenced from your website's HTML page using a manifest link. Insert the following code in between the `<head>` and `</head>` tags of your website's HTML page to link to your manifest file:
+
+```html
+<link rel="manifest" href="/manifest.json">
+```
+
+> [!NOTE]
+> The manifest file's content must be valid JSON, but the file can also be named like `app_name.webmanifest`. If you do choose to use the `webmanifest` extension, verify that your HTTP server serves it with the `application/manifest+json` MIME type.
+
+At a minimum, a manifest file should contain the following information:
+
+```json
+{
+    "lang": "en-US",
+    "name": "My Sample PWA",
+    "short_name": "SamplePWA",
+    "description": "A sample PWA for testing purposes"
```

Comment:
I’d say `name`, `lang` & `start_url` are a good baseline. Description has moved to the App Info spec.